modules/099-ceph-csi/docs/USAGE.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "The ceph-csi module: usage"
+---
+
+## An example of the `CephCSI` configuration
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: CephCSI
+metadata:
+  name: example
+spec:
+  clusterID: 2bf085fc-5119-404f-bb19-820ca6a1b07e
+  monitors:
+  - 10.0.0.10:6789
+  userID: admin
+  userKey: AQDbc7phl+eeGRAAaWL9y71mnUiRHKRFOWMPCQ==
```

Comment:
Is this encrypted? Looks like a plain text keyring to me.

Reply:
No, it's a value obtained with `ceph auth get client.admin` . Base64 contains binary data.

Reply:
Data in this object looks too sensitive to leave it there in plain text. What do you think?

Reply:
If we place them in a dedicated secret, then it will not become safer.